omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/SendPatientToQueueFragmentController.java

```diff
@@ -0,0 +1,64 @@
+package org.openmrs.module.patientqueueing.fragment.controller;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.Provider;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.module.patientqueueing.api.PatientQueueingService;
+import org.openmrs.module.patientqueueing.model.PatientQueue;
+import org.openmrs.ui.framework.SimpleObject;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.openmrs.ui.framework.fragment.FragmentModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+import java.util.Date;
+
+public class SendPatientToQueueFragmentController {
+
+    private final Logger log = LoggerFactory.getLogger(SendPatientToQueueFragmentController.class);
```

Comment:
Did you just forget how we log? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging